requirements.txt

```diff
@@ -1,5 +1,5 @@
-flake8==2.5.4
+flake8==2.6.0
 mccabe==0.4.0
 pep8==1.7.0
 pyflakes==1.0.0
-flake8-print==2.0.2
\ No newline at end of file
+flake8-print==2.0.2
```

Comment:
We should separate test requirements into a separate file (test_requirements.txt)